python/chemiscope/structures/_mda.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+
+try:
+    import MDAnalysis as mda
+
+    HAVE_MDA = True
+except ImportError:
+    HAVE_MDA = False
+
+
+def _mda_valid_structures(frames):
+    if HAVE_MDA and isinstance(frames, mda.Universe):
+        return frames, True
+    else:
+        return [], False
+
+
+def _mda_to_json(ag):
+    data = {}
+    data["size"] = len(ag)
+    data["names"] = [atom.name for atom in ag]
```

Comment:
Is this the atom name or element? i.e. `CA` or `C` for alpha carbons? From chemiscope perspective, this should be the element name rather than the atom type.

Reply:
Here this is the atom name, `CA` for alpha carbons. Yes I think the element name makes more sense.